src/components/Watchlater/Watchlater.jsx

```diff
@@ -1,7 +1,55 @@
-import React from "react";
+import React, { useEffect, useState } from "react";
+import { useNavigate } from "react-router-dom";
+import axios from "axios";
+import { useVideo } from "../../context";
+import { WatchlaterCard } from "../WatchlaterCard/WatchlaterCard";
```

Comment:
It can be reduced and directly imported from the index of components.

Reply:
this is with in the component folder so it will import from the component directly